packages/babel-preset-jsx/package.json

```diff
@@ -8,6 +8,7 @@
   "author": "Nick Messing <dot.nick.dot.messing@gmail.com>",
   "license": "MIT",
   "private": false,
+  "files": [],
```

Comment:
The `module` field won't be included by default so `files` can't be left empty.

Reply:
Ah, right, I forgot there's `module` too.